devportal/devportal-domain-service/ballerina/Dependencies.toml

```diff
@@ -121,7 +121,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.4.0"
+version = "1.4.1"
```

Comment:
add the same change for template file

Reply:
Added from da7e3b8283a1e3da884958a332c7da14467b9f14